### ISSUE TITLE
Not an issue by the way but an admiration for your genius

### ISSUE THREAD

**Issue by shellking4** [jmcdo29/nestjs-dynamic-render#1]
I really like the solution. I realllllllly like it. Thank you keep it up. Looking forward to becoming a great engineer as you.

**Comment by jmcdo29** [comment]
Thank you :pray: It's been a while since I looked at this solution or even remembered what it would be used for. Always nice to look back at things like this. Keep at it, you'll get there. :+1:

**Comment by shellking4** [comment]
Thank you very much. I'll keep working
An example of use case for this: I have a project where I need a rest api, a web interface and a mobile interface. Page reloading at each request is not a problem for me now, so rather than investing time in building a frontend with react or angular or vue or svelte, I can keep everything web and api related in Nestjs. Then when I call the endpoints in the web browser, using your method I got html pages and from my flutter mobile app, calling the same endpoints I got the api response with the data I need. It really solves some complexity and logic duplication issues. This is why I got very excited about it :))
